app/controllers/test_messages_controller.rb

```diff
@@ -0,0 +1,25 @@
+class TestMessagesController < ApplicationController
+  def new
+    @message = ScheduledMessage.find(params[:scheduled_message_id])
+  end
+
+  def create
+    message = ScheduledMessage.find(params[:scheduled_message_id])
+    employee = Employee.find_by(slack_username: slack_username)
+
+    if employee
+      MessageSender.new(employee, message).delay.run
```

Comment:
YES just got it to work - realized the reason it wasn't authing was that we were configuring the Slack client in `initialize`...so when we called `delay`, the Slack client wasn't working because there was a disconnect between the `initialize` and the `run` - moved `configure_slack` into `run` and now IT WORKS